code/modules/spacepods/spacepod.dm

```diff
@@ -204,6 +204,7 @@
 	deal_damage(30)
 
 /obj/spacepod/attack_animal(mob/living/simple_animal/user)
```

Comment:
This is a perfect example of why spacepods are awfully coded

Reply:
Don't tempt me to replace spacepods with like.... an exploration mech.... and eliminate all spacepod code.